MPF.Frontend/Tools/SubmissionGenerator.cs

```diff
@@ -417,7 +449,22 @@ private static SubmissionInfo CreateDefaultSubmissionInfo(BaseProcessor processo
                 return driveLabel;
             }
 
-            // If only one label, don't mention fs
+            // Remove duplicate/useless volume labels
+            string? defaultLabel = SimplifyVolumeLabel(driveLabel);
+            if (defaultLabel != null && defaultLabel.Length != 0)
```

Comment:
So you're only going to remove duplicates if the drive label could be simplified? So if there's no drive label (for some reason) but there are other labels, then those are okay to append even if they're the same?

Reply:
I've made it such that if drive label doesn't exist (and Linq is available) then it will try find the UDF label to act as the default.

Reply:
We'll see how this ends up working :+1: